NetKet/GroundState/variational_montecarlo.hpp

```diff
@@ -452,13 +512,17 @@ class VariationalMonteCarlo {
     MPI_Barrier(MPI_COMM_WORLD);
   }
 
-  void setSrParameters(double diagshift = 0.01, bool rescale_shift = false,
+  void SetSrParameters(double diag_shift = 0.01, bool rescale_shift = false,
                        bool use_iterative = false, bool use_cholesky = true) {
```

Comment:
Maybe we can change the interface here (and in the VMC constructor) to take a `StochasticReconfiguration::Solver` value directly instead of having the two flags `use_iterative` and `use_cholesky`?